jsk_2016_01_baxter_apc/euslisp/examples/picking-with-clustering.l

```diff
@@ -22,6 +22,8 @@
   (send *ri* :wait-interpolation)
   (send *ri* :stop-grasp *arm*)
   (send *ri* :spin-off-by-wrist *arm* :times 20)
+  (send *ri* :gripper-servo-on *arm*)
+  ;; power on gripper servo as it was powered off in :move-arm-body->order-bin
```

Comment:
Please update the comment: `power on gripper servo as it was powered off in :move-arm-body->order-bin` -> `power on gripper servo`